docker-compose-example/.env

```diff
@@ -1,17 +1,17 @@
 PUID=1000
 PGID=1000
-ARM64_DEVICE=adlink
+ARM64_DEVICE=generic
 USE_DEPOT_DOWNLOADER=false
 WORLD_INDEX=0
 WORLD_NAME="Core Keeper Server"
 WORLD_SEED=0
 WORLD_MODE=0
 GAME_ID=""
-DATA_PATH="${STEAMAPPDATADIR}"
+DATA_PATH="${STEAMAPPDATADIR:-/home/steam/core-keeper-data}"
 MAX_PLAYERS=10
 SEASON=""
 SERVER_IP=""
-SERVER_PORT=""
+SERVER_PORT="27015"
```

Comment:
This might change the behavior of the server on start, as seen in the de-compiled dlls.
```cs
if (CommandLineArgs.Has("-port"))
{
	if (!ushort.TryParse(CommandLineArgs.GetParam("-port"), out num3))
	{
		num3 = 0;
	}
	if (CommandLineArgs.Has("-ip"))
	{
		text2 = CommandLineArgs.GetParam("-ip");
	}
	else
	{
		text2 = "0.0.0.0";
	}
}
if (num3 != 0)
{
	IPAddress ipaddress;
	if (text2 != null && IPAddress.TryParse(text2, out ipaddress))
	{
		steamServerInit.IpAddress = ipaddress;
	}
	steamServerInit.GamePort = num3;
	steamServerInit.QueryPort = steamServerInit.GamePort + 1;
	Debug.Log(string.Format("Steam server using game port: {0} query port: {1} address: {2}", steamServerInit.GamePort, steamServerInit.QueryPort, steamServerInit.IpAddress));
}
else
{
	steamServerInit.IpAddress = IPAddress.Loopback;
	steamServerInit.WithQueryShareGamePort();
}
```

Reply:
If i remember correctly, setting the port will make the server start in direct connection mode, meaning clients will try to connect directly to the server and that port has to be open for connections, otherwise connections will fail.

If its not set it will start in steam relay mode, meaning no ports have to be open, since all connections will be relayed thru steam relay network.

But if i am wrong, it means we can start the server with the server query functioning, that open an possibility for an proper healthcheck

Reply:
I expose it there so the docker compose can open them. AFAIK, without exposing them, the docker compose container won't be reachable.